api/users/pagination.py

```diff
@@ -0,0 +1,49 @@
+from collections import OrderedDict
+
+from api.base.pagination import JSONAPIPagination
+
+
+class UserLogPagination(JSONAPIPagination):
+
+    node_log_aggregates = None
+
+    def get_response_dict(self, data, url):
+        if self.request.query_params.get('aggregates'):
+            comments, nodes, wiki, files = self.node_log_aggregates()
+            response_dict = OrderedDict([
```

Comment:
You could also call super(UserLogPagination, self).get_response_dict(data, url) here and then add the `meta` object to it.

Or call super() once in  the beginning, return that if no 'aggregates' query, otherwise tack the meta object onto the response_dict.